package/Runtime/Scripts/Internal/NimbusAdUnit.cs

```diff
@@ -8,38 +7,35 @@ namespace Nimbus.Runtime.Scripts.Internal {
 	public sealed class NimbusAdUnit {
 		private readonly AdEvents _adEvents;
 		public readonly AdUnityType AdType;
-
-		// Delay before close button is shown in milliseconds, set to max value to only show close button after video completion
-		// where setting a value higher than the video length forces the x to show up at the end of the video
-		internal readonly int CloseButtonDelayMillis;
+		
 		public readonly int InstanceID;
 		public readonly string Position;
-		
+		public MetaData ResponseMetaData;
+
 		internal AdError AdControllerError;
 		internal AdError AdListenerError;
 		internal bool AdWasRendered;
 		internal BidFloors BidFloors;
 		internal AdEventTypes CurrentAdState;
-		internal MetaData MetaData;
 		
+		// Delay before close button is shown in milliseconds, set to max value to only show close button after video completion
+		// where setting a value higher than the video length forces the x to show up at the end of the video
+		internal readonly int CloseButtonDelayMillis;
```

Comment:
Let's change this to seconds unless you feel millis would be better. I added a new method that will take in seconds as a parameter in the Android SDK.

Reply:
@JasonSznol this is for IOS and Bruno created it, leave it for now.

Reply:
I'll leave this as is for now. Since it's internal it really makes no difference.